extensions/hp-extension/src/hp/hpHeart.ts

```diff
@@ -245,11 +230,14 @@ export default {
         {
           viewportOptions: {
             toolGroupId: 'default',
-            customViewportOptions: {
-              zoomPan: {
-                initialDisplayArea: [0.4, 0.4],
+            syncGroups: [
+              {
+                type: 'initialZoomPan',
+                options: { 
+                  initialDisplayArea: [0.4, 0.4],
```

Comment:
there is a inconsistence on your linter. Some places like here there is a space. Some other there is not. Can you ensure your linter or project linter runs on all files?